### ISSUE TITLE
Jirassic command line - start command

### ISSUE THREAD

**Issue by trebizonder** [ralcr/Jirassic#115]
Hello,

I've tried to start a day via the command line with the command `jirassic start`, here is the message that is displayed : ' Unsupported command. '.

My config :  Macbook Pro  - macOS Mojave.

Could you please help me to start a day ?, since any operation require to start a day, that's a blocking issue

**Comment by ralcr** [comment]
Hi, did you install the command line? It must be installed manually, it's a separate app, i have instructions on the website http://www.jirassic.com

$ sudo curl -o /usr/local/bin/jirassic https://raw.githubusercontent.com/ralcr/Jirassic/master/build/jirassic
$ sudo chmod +x /usr/local/bin/jirassic


Didn't tested in a while, hope is still working

**Comment by julianwki** [comment]
I've got the same problem. What's also weird is that small exclamation mark stating "some plugins need updating.", see screenshot. Something not correct here...
<img width="695" alt="Screenshot 2020-08-06 at 19 28 10" src="https://user-images.githubusercontent.com/617179/89563104-3863c280-d81b-11ea-95ea-d75ed14c679a.png">

My config:
- MacBook Pro
- macOS Catalina
- Shell scripts 1.0 installed
- Jirassic 18.12.12 installed

**Comment by mghweb** [comment]
Yeah, I can confirm that the `jirassic start` command does not exist in the current CLI version. Is there an older version I can install?
